components/api_server/src/initialization/database.py

```diff
@@ -26,7 +26,7 @@ def init_database() -> Database:  # pragma: no feature-test-cover
     import_datamodel(database)
     initialize_secrets(database)
     initialize_reports_overview(database)
-    if os.environ.get("LOAD_EXAMPLE_REPORTS", "True").lower() == "true":
+    if getenv("LOAD_EXAMPLE_REPORTS").lower() == "true":
```

Comment:
Why not explicit bool cast?

Reply:
`getenv` (note this is our own `getenv`, not `os.getenv`) always returns a string.

Reply:
I meant, `if bool(getenv(..)):`